packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.test.tsx

```diff
@@ -0,0 +1,318 @@
+import { Collapse as MockCollapse } from '@mui/material'
```

Comment:
Copied RangeFacetFilter tests, and modified to have 2 faceted columns and test for the expected callback values array